javascript/implementation/CommandLineInterface.ts

```diff
@@ -64,8 +63,8 @@ export class CommandLineInterface {
             logToStdErr(`using data root ${dataRoot}`);
             ensure(process.env["GINK_PORT"]);
         } else if (dataFile) {
-            logToStdErr(`using data file=${dataFile}, reset=${reset}`);
-            this.store = new LogBackedStore(dataFile, reset);
+            logToStdErr(`using data file=${dataFile}`);
```

Comment:
This is the only non-documentation change. I'm going to redo this interface in a following PR anyway, but I am removing it here because it doesn't work.